neuro_san/http_sidecar/handlers/connectivity_handler.py

```diff
@@ -49,3 +49,9 @@ async def get(self, agent_name: str):
         finally:
             await self.flush()
             self.logger.info(metadata, "Finish GET %s/connectivity request", agent_name)
+
+    async def options(self):
+        """
+        Implementation of OPTIONS request handler for streaming chat API call.
+        """
+        await self.flush()
```

Comment:
FWIW: This is the same thing that happens with streaming_chat()

Reply:
Do these also need to happen in concierge_handler.py, health_check_handler.py and openapi_publish_handler.py?

Reply:
I stole it from `streaming_chat`. 

The only time we care about CORS is with browser-to-service interactions and the browser app doesn't hit those other endpoint you mentioned. I guess it might some day? Should the `OPTIONS` function be in the base handler?

Reply:
`concierge_handler.py` is the `/list` endpoint I mentioned. Somehow it's working fine without `OPTIONS` support - maybe due to the "simple request" rule I mentioned in the description.